en/documentation/assets.md

```diff
@@ -21,13 +23,13 @@ section_id: documentation
       <div class="row">
         <div class="statement-buttons">
           <div class="statement-button">
-            {% capture file_url %} {{ site.url }}{{ site.baseurl }}/files/icons/{{ button }}.dark-white-interior-blue-type{% endcapture %}
+            {% capture file_url %}{{ site.url }}{{ site.baseurl }}/files/buttons/{{ button }}.dark-white-interior-blue-type{% endcapture %}
```

Comment:
Does this change and the one below, along with the renaming of the files, break the templates for the right statements used by the rights-app?